baeksehee/src/pages/_app.tsx

```diff
@@ -0,0 +1,47 @@
+import type { AppProps } from "next/app";
+import Link from "next/link";
+import styled from "styled-components";
+import GlobalStyle from "../styles/GlobalStyle";
+
+export default function App({ Component, pageProps }: AppProps) {
+  return (
+    <>
+      <GlobalStyle />
+      <StyledBody>
+        <nav>
+          <StyledImg src="../megabox.png"></StyledImg>
+
+          <p>
+            <Link href="/">
+              <StyledSpan>상영 영화 </StyledSpan>
+            </Link>
+            <Link href="/booking">
+              <StyledSpan>예매하기</StyledSpan>
+            </Link>
+          </p>
+        </nav>
+      </StyledBody>
+
+      <Component {...pageProps} />
+    </>
+  );
+}
+
+const StyledBody = styled.body`
+  margin: 0;
+  padding-top: 20px;
+
+  padding-bottom: 50px;
+`;
```

Comment:
지금 여기서 에러가 발생하는거에요 ~~

`body` 태그는 사용하시면 안됩니다 ~
```suggestion
const StyledBody = styled.div`
  margin: 0;
  padding-top: 20px;

  padding-bottom: 50px;
`;
```

자세하게 발생한 이유에 대해 설명하자면, Next.js의 하이드레이션이라는 과정에 대해 알면 되는데요.

서버 사이드에서 만들어진 파일에 클라이언트 사이드 동작을 추가하는 (물을 채우는 = 하이드레이션) 과정에서 body라는 태그가 두개라서 발생하는 거 같아요

Reply:
!😁자세한 이유  알려주셔서 감사합니다!